Snail/Snail+Combine/SnailSubscription.swift

```diff
@@ -0,0 +1,34 @@
+//  Copyright © 2021 Compass. All rights reserved.
+
+#if canImport(Combine)
+import Combine
+import Foundation
+
+@available(iOS 13.0, *)
+class SnailSubscription<Upstream: ObservableType, Downstream: Combine.Subscriber>: Combine.Subscription where Downstream.Input == Upstream.T, Downstream.Failure == Error {
+    private var disposable: Subscriber<Upstream.T>?
```

Comment:
can this be `let`?

Reply:
As of now, no. I'm setting it to `nil` [here](https://github.com/UrbanCompass/Snail/pull/159/files#diff-0927c7172ad6dc5a647d86f49df38916c696fabe5e531acfc72cb86fedd8aaf3R31).